src/ontology/edits/oeo-social.omn

```diff
@@ -1093,6 +1093,116 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/956",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "capital investment cost",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "fixed investment cost",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "investment",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "FIBO definition of investment relates to property only: https://spec.edmcouncil.org/fibo/ontology?searchBoxQuery=investment, 
+
+Gabler on Investition: langfristige Bindung finanzieller Mittel in materiellen oder in immateriellen Vermögensgegenständen (https://wirtschaftslexikon.gabler.de/definition/investition-39454)",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1022",
+        rdfs:label "investment cost"
+    
+    SubClassOf: 
+        OEO_00040009
+    
+    
+Class: OEO_00020168
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000112> "rent, regular technical maintenance cost, staff cost, certification cost",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Fixed costs are costs for operation and mantainance that do not vary with the amount of goods or services produced.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "fixed operation and maintanance cost",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "oemof-Model: https://oemof-solph.readthedocs.io/en/latest/_modules/oemof/solph/options.html?highlight=fix%20costs
+
+Fixe Kosten: Kosten, die von der jeweils betrachteten Einflussgröße bzw. Entscheidung unabhängig sind, d.h. Kosten, die sich nicht mit der jeweils betrachteten Einflussgröße ändern. (https://wirtschaftslexikon.gabler.de/definition/fixe-kosten-33359)",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1022",
+        rdfs:label "fixed cost"
+    
+    SubClassOf: 
+        OEO_00040009
+    
+    
+Class: OEO_00020169
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Equipment costs are investment costs for acquisition of the components and machinery including environmental facilities.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1022",
+        rdfs:label "equipment cost"
+    
+    SubClassOf: 
+        OEO_00020167
+    
+    
+Class: OEO_00020170
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000112> "costs that occur for becoming operational",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Installation and development cost are investment costs incurred for setting up the installation.",
```

Comment:
The definition reads _installation and development cost_ while the label is only _installation cost_. Is that intended?

Reply:
I added `installation and development cost` as alternative term